taxi.proto

```diff
@@ -15,9 +15,18 @@ service Admin {
   rpc ListTopups(ListTopupsRequest) returns (ListTopupsReply);
   // ChangeSpread updates the percentage taken as service fee
   rpc ChangeSpread(ChangeSpreadRequest) returns (ChangeSpreadReply);
+  // AddAsset lets to add a new asset to support
+  rpc AddAsset(AddAssetRequest) returns (AddAssetReply);
 }
 
+message AddAssetRequest {
+  string asset_hash = 1;
+  string ticker = 2;
```

Comment:
I would add also the price, otherwise how it can be set?

Reply:
the ticker will be used to fetch the price (from external APIs)

Reply:
Finally I just add the method UpdateOrCreatePrice: basically that's exactly an "TDEX's UpdateMarketPrice" ---> then the feeder will use the RPC instead of integrating feeder as package in Taxi (not very maintainable or scalable). What do you think ? @altafan @tiero